Core/Blob/Extensions/BlobServiceBuilderExtensions.cs

```diff
@@ -38,12 +39,37 @@ public static class BlobServiceBuilderExtensions
 
             builder
                 .AddProgramInitializer<BlobInitializerService>()
-                .AddAssemblies(typeof(AutoMapperProfile).Assembly)
+                .AddAssemblies(typeof(Data.AutoMapperProfile).Assembly)
```

Comment:
I don't think you need the new Data., since the directive Lens.Core.Blob.Data is already used.

Reply:
![image](https://user-images.githubusercontent.com/367462/180294794-8c11f928-8982-48c6-9820-cea882666fbc.png)